src/Map.tsx

```diff
@@ -1,24 +1,20 @@
-import React, { RefObject, useEffect, useRef } from 'react'
+import React, { useEffect, useRef } from 'react'
 import L, { LeafletEventHandlerFnMap, MapOptions } from 'leaflet'
-import MapContext from './MapContext'
+import MapContext, { MapInstance } from './MapContext'
 import useLeafletEvents from './utils/useLeafletEvents'
-import { AllLeafletInstances } from './types'
 
 type Props = {
   events?: LeafletEventHandlerFnMap
   options?: MapOptions
 }
 
 const Map = React.forwardRef<
-  RefObject<HTMLDivElement>,
+  HTMLDivElement,
   React.HTMLProps<HTMLElement> & Props
 >(({ children, events, options, ...otherProps }, ref) => {
-  const [
-    mapInstance,
-    setMapInstance,
-  ] = React.useState<AllLeafletInstances | null>(null)
+  const [mapInstance, setMapInstance] = React.useState<MapInstance>(null)
```

Comment:
Should we import `useState` as the other hooks? So without the `React` namespace.